### PR TITLE
Fix up Gemfile

### PR DESCRIPTION
**Pull request by gabrielle27**
@comike011 let me know what you think of this solution. 

tldr;

I upgraded jekyll to the latest which allows us to completely get rid of yajl-ruby. 

---

Previously this PR https://github.com/lvhdev/lifeio-website/pull/28
brought in an update to our yajl-ruby gem to fix a
security vulnerability

I believe this introduced an error because pygments.rb (0.6.3) depended on the
older vulnerable version of yajl-ruby and jekyll (2.5.3) depended on pygments.rb (0.6.3)

I updated jekyll to the latest version and it no longer depends on pygments.rb so we no longer need yajl-ruby.

After this change I had no problem installing all gems from a clean project and starting my server. Site looks good in Chrome and Firefox.

### CODE REVIEW

**Comment by comike011** [comment]
lgtm

**Comment by gabrielle27** [comment]
@comike011 thankyou!!
